### PR TITLE
Add new GenerateJLIClassesHelper stub method

### LINKED ISSUE #1606: JDK11 build 11+7 fails missing new GenerateJLIClassesHelper method
Merge of latest 11+7 level fails with missing method in GenerateJLIClassesHelper:
byte[] generateInvokersHolderClassBytes(String className,
                    MethodType[] invokerMethodTypes,
                    MethodType[] callSiteMethodTypes)
Causes compile failure in 
src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java at
                return GenerateJLIClassesHelper
                        .generateInvokersHolderClassBytes(className,
                                invokerMethodTypes, callSiteMethodTypes);

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Fixes #1606 

Signed-off-by: Andrew Leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

**Comment by andrew-m-leonard** [comment]
Fixes #1606

File: jcl/src/java.base/share/classes/java/lang/invoke/GenerateJLIClassesHelper.java [hunk 1]
```diff
@@ -49,4 +49,8 @@
 	static byte[] generateInvokersHolderClassBytes(String str, MethodType[] mts) {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+
+        static byte[] generateInvokersHolderClassBytes(String str, MethodType[] invokerMts, MethodType[] callMts) {
```
Comment (JasonFengJ9) [inline jcl/src/java.base/share/classes/java/lang/invoke/GenerateJLIClassesHelper.java]: Is this method only required by `Java 11`? If so, pls decorate it with `JPP` flag accordingly.

Comment (DanHeidinga) [reply jcl/src/java.base/share/classes/java/lang/invoke/GenerateJLIClassesHelper.java]: ie wrap the method with:
```
/*[INCLUDE-IF Java11]*/

/*[ENDIF]*/
```

Comment (andrew-m-leonard) [reply jcl/src/java.base/share/classes/java/lang/invoke/GenerateJLIClassesHelper.java]: Done this now

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

**Comment by pshipton** [comment]
jenkins compile plinux jdk9

**Pull request merged**